localstack/services/secretsmanager/provider.py

```diff
@@ -495,7 +495,22 @@ def backend_update_secret_version_stage(
 
 @patch(FakeSecret.reset_default_version)
 def fake_secret_reset_default_version(fn, self, secret_version, version_id):
-    fn(self, secret_version, version_id)
+    # fn(self, secret_version, version_id)
+    # FIXME: remove for the next moto bump (above 4.2.post1) and uncomment line above
+    # remove all old AWSPREVIOUS stages
```

Comment:
Completely unrelated and task for futures us:
Would be great if we can find a way to unify this kind of thing across the code base a bit. In a lot of cases this is just forgotten and then causes issues down the road when upstream changes.

Reply:
I could also imagine adding a unit test that will fail as soon as we update moto, so that we are automatically reminded to fix this :thinking: 

again: not really something that needs any action right now